### PR TITLE
docs: Add troubleshooting note for Neo4j docker container on Windows

### PR DESCRIPTION
**Pull request by prafullkotecha**
### Tests

Additional test not needed, since the changes are done to documentation only.

### Documentation

- Add troubleshooting note for Neo4j docker container on Windows

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes.
- [x] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain docstrings that explain what it does

### CODE REVIEW

**Review by dorianj — APPROVED** [review]
LGTM pending DCO. Thank you!

File: .gitignore [hunk 1]
```diff
@@ -51,4 +52,4 @@ coverage
 
 # Helm
 amundsen-kube-helm/templates/helm/charts/*
-amundsen-kube-helm/templates/helm/requirements.lock
+amundsen-kube-helm/templates/helm/requirements.lock
```
Comment (dorianj) [inline .gitignore:55]: oops?

**Comment by feng-tao** [comment]
why the commit list is that long?

**Review by feng-tao — CHANGES_REQUESTED** [review]
why the commit list is that long? could we rebase it with latest master?

**Comment by prafullkotecha** [comment]
> why the commit list is that long? could we rebase it with latest master?

I am going to close this PR and open a new one. With latest code from `main` branch.

**Pull request closed without merging**